gns3server/templates/homepage.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head> 
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/> 
+<title>GNS3 Server</title>
+</head>
+<body>
+{% block body %}{% endblock %}
+</body>
+<h3>
+    Welcome to Home page of GNS3.
+</h3>
+<h4>
+    Visit the upload page to try GNS3 features: <a href = "http://127.0.0.1:8000/upload">Upload</a>
```

Comment:
@noplay True that. I have updated the pr and squashed the commits. Kindly review it.